src/main/java/com/awesometeam/Invoicify/invoice/model/Items.java

```diff
@@ -35,4 +32,52 @@ public Items(String description, char feeType, int quantity, Double fee, Double
         this.fee = fee;
         this.amount = amount;
     }
+
+    public long getId() {
```

Comment:
While doing refactoring please change to use Lombok instead or explicit getter/setter